### ISSUE TITLE
Further alignment of structure/wording of 2.5.5 Target Size (Enhanced) and 2.5.8 Target Size (Minimum)

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#3086]
There's currently already an issue about aligning the "Inline" definition https://github.com/w3c/wcag/issues/2857 but I'd suggest going further and making sure the structure and wording of the two SCs are not unnecessarily different.

* 2.5.8 Target Size (Minimum) introduces the exceptions with "except *where*:" and then has an *unordered list* for the exceptions; 2.5.5 Target Size (Enhanced) introduces the exceptions with "except *when*:" and then uses a *description list* for the exceptions
* 2.5.8 has the following definition/exception: "Equivalent: The function can be achieved through a different control on the same page that meets this criterion."; 2.5.5 uses different wording to mean the same? "Equivalent: The target is available through an equivalent link or control on the same page that is at least 44 by 44 CSS pixels;"
* 2.5.8 has the following definition/exception: "Essential: A particular presentation of the target is [essential](https://w3c.github.io/wcag/guidelines/22/#dfn-essential) or is legally required for the information being conveyed;"; 2.5.5 has "Essential: A particular presentation of the target is [essential](https://w3c.github.io/wcag/guidelines/22/#dfn-essential) to the information being conveyed." ... so, is the "legally required" aspect of no concern for AAA ? And if so, why? Or just an oversight? (this leaves me with a weird "can't do AAA and be legal" feeling.

I'd say pick one of these as the authoritative wording and format, and stick with it consistently. And then harmonise the exceptions that are meant to be the same, and use the same consistent wording for both the AA and AAA version

**Comment by JAWS-test** [comment]
> 2.5.8 has the following definition/exception: "Equivalent: The function can be achieved through a different control on the same page that meets this criterion."; 2.5.5 uses different wording to mean the same? "Equivalent: The target is available through an equivalent link or control on the same page that is at least 44 by 44 CSS pixels;"

This is an important substantive difference that is probably intentional and should be kept:
* for AA, it is also sufficient if the equivalent target satisfies the SC (incl. the exceptions, e.g. inline, spacing, etc.).
* for AAA, the equivalent target must really be sufficiently large and must not use any exception (inline, user agent control etc.)

If not intended, then it should be adjusted, but 2.5.5 can't be adjusted anymore...? Pity, because an adjustment at 2.5.5 would make more sense than at 2.5.8

**Comment by JAWS-test** [comment]
> 2.5.8 has the following definition/exception: "Essential: A particular presentation of the target is [essential](https://w3c.github.io/wcag/guidelines/22/#dfn-essential) or is legally required for the information being conveyed;"; 2.5.5 has "Essential: A particular presentation of the target is [essential](https://w3c.github.io/wcag/guidelines/22/#dfn-essential) to the information being conveyed." ... so, is the "legally required" aspect of no concern for AAA ? And if so, why? Or just an oversight? (this leaves me with a weird "can't do AAA and be legal" feeling.

This is an important substantive difference that is probably intentional and should be kept:
* for AA someone can refer to legal requirements (the form should look like in paper form)
* for AAA one cannot refer to legal requirements, because here at this level the accessibility is more important than the legal requirements.

This distinction seems to make sense to me

**Comment by patrickhlauke** [comment]
> This is an important substantive difference that is probably intentional

sorry @JAWS-test ... are you part of the working group? there's no need for "probably intentional" second guessing. I'm asking fellow working group members here...

**Comment by mbgower** [comment]
@patrickhlauke we need to make sure we are careful with the wording "overlap". It might be okay, but we need to make sure we use them synonymously and appropriately (based on where the normative language goes)

**Comment by patrickhlauke** [comment]
@mbgower as @alastc is tackling this in https://github.com/w3c/wcag/pull/3097 this comment is probably more aimed at him. i'm happy either way as long as the two are not unnecessarily divergent

**Comment by mbgower** [comment]
Right, my point was that once the normative is complete, the language here should be assessed to make sure it complements. With 'overlap' removed from the normative text, I think it's less of a problem, in any case.

**Comment by patrickhlauke** [comment]
need to work out relationship with https://github.com/w3c/wcag/pull/3123

**Comment by mbgower** [comment]
@WilcoFiers as per your comments on the call today regarding #3097, I'd like to better understand why you think we had already addressed this issue. I don't recall anyone making the case that the other exceptions shouldn't be allowed where a target is not 48x48.
I'm not opposed to that, but I don't feel like we had the discussion in, say, the past year? It seems to me it's been some time since we looked at AAA, and the AA landscape has changed quite a bit.

**Comment by patrickhlauke** [comment]
@alastc although this is closed, I think this aspect has not been addressed:

> 2.5.8 Target Size (Minimum) introduces the exceptions with "except where:" and then has an unordered list for the exceptions; 2.5.5 Target Size (Enhanced) introduces the exceptions with "except when:" and then uses a description list for the exceptions

**Comment by patrickhlauke** [comment]
I've now filed https://github.com/w3c/wcag/pull/3189 to address these missed-out aspects that weren't covered by #3123
